frontend/src/app/components/RetireConfirmationDialog/index.js

```diff
@@ -37,9 +35,9 @@ export class RetireConfirmationDialog extends Component<RetireConfirmationDialog
               <Localized id="retireMessageUpdate">
                 <p>As you wish. This will uninstall Test Pilot. You can disable individual experiments from the Firefox Add-ons Manager.</p>
               </Localized>
-              <LocalizedHtml id="retireEmailMessage">
+              <Localized id="retireEmailMessage" em={<em></em>}>
```

Comment:
In future version of `fluent-react`, translations will be able to use text-level elements like `<em>` and `<br>` without requiring the code to pass them as props explicitly. See https://github.com/projectfluent/fluent.js/issues/186. Once this lands, I'll file another PR to remove all these `em={<em></em>}` et al.